lightning-network-book.tex

```diff
@@ -100,16 +100,16 @@
   description={}}
 \newglossaryentry{script}{name={Bitcoin Script},
   description={}}
-\newglossaryentry{msat}{name={millisatoshi},
+\newglossaryentry{msat}{name={Millisatoshi},
```

Comment:
I had millisatoshi small on purpose since milli is usually a small m and Mega is usually a large one

Reply:
ACK